site/content/docs/_index.md

```diff
@@ -65,7 +65,7 @@ monitoring in minutes.
 {{< doc-card
 	  class="two-column-card"
 	  headerTag="h3"
-	  title="Create a multistep API check"
+	  title="Create a multistep check"
```

Comment:
```suggestion
	  title="Create a Multistep check"
```

Reply:
Scrap that, I talked to Sara & she agreed that we should use upper case. I'll change it for the other checks as well on this page.